src/rust/engine/Cargo.toml

```diff
@@ -115,6 +123,12 @@ url = "2.1"
 uuid = { version = "0.7", features = ["v4"] }
 workunit_store = { path = "workunit_store" }
 
+[dev-dependencies]
+testutil = { path = "./testutil" }
+fs = { path = "./fs" }
+tokio = "0.1"
```

Comment:
Ah. This would probably do it.

Reply:
Should this be tokio = 0.2 then?

Reply:
Ah, I think I can just use tokio from the the engine dependencies now.